README.md

```diff
@@ -1,12 +1,13 @@
 # Summary
 
-This docker-compose is to start an instance of S64 DA or PSQL or both.
-
+Swarm64 DA Docker Compose will assist to start instance of S64 DA or native PSQL.
+Once the instance is up and running you can connect through a psql client or run
+a benchmark using the [Swarm64 TPC Toolkit](https://github.com/swarm64/tpc-toolkit).
 
 # Prerequisites
 
 - Python 3.6 or higher
-- docker-compose
+- docker-compose (eg. use: python3.6 -m pip install docker-compose)
```

Comment:
That command is not necessarily available everywhere, could also be just `python3` IMO.

Reply:
Added python3.6 / python3